source/pixform.h

```diff
@@ -159,6 +84,8 @@
 #define MAX_RED_D(F)            CONCAT(MAX_RED_,F)
 #define MAX_BLUE_D(F)           CONCAT(MAX_BLUE_,F)
 #define MAX_GREEN_D(F)          CONCAT(MAX_GREEN_,F)
+#define RED_SHIFT_BITS_D(F)     CONCAT(RED_SHIFT_BITS_, F)
+#define GREEN_SHIFT_BITS_D(F)   CONCAT(GREEN_SHIFT_BITS_, F)
```

Comment:
I think you also need to add a guard for PSP here, or otherwise it will break compilation.

Reply:
Actually, it doesn't (I did test compile the PSP build...)

But for correctness, I believe that you are right. I'll add the guards.